### PR TITLE
Use 'using namespace dealii' within StepXX namespace in tutorials

### PR DESCRIPTION
**Pull request by AE7TB99**
This PR addresses issue #17249 by refactoring the namespace usage in the tutorial programs to consistently import the `dealii` namespace within the `StepXX` namespace.

### CODE REVIEW

File: doc/news/changes/minor/20240728AE7TB99 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Uniformly apply the practice of importing the 'dealii' namespace
+only within the StepXX namespace across all tutorial programs.
+<br>
+(Lóránt Hadnagy, 2024/07/28)
```
Comment (peterrum) [inline doc/news/changes/minor/20240728AE7TB99:4]:
```suggestion
(Hadnagy Lóránt, 2024/07/28)
```

;)

**Review by peterrum — COMMENTED** [review]
Looks good. But could you revert the formatting changes?

File: doc/news/changes/minor/20240728AE7TB99 (hunk 1 above)
Comment (AE7TB99) [reply doc/news/changes/minor/20240728AE7TB99]: Should I change it back then? It doesn't matter to me, but I want it to be consistent ^^

**Comment by AE7TB99** [comment]
Indent check still does not pass, but I'm working on it...
Thank you for your patience.

File: doc/news/changes/minor/20240728AE7TB99 (hunk 1 above)
Comment (AE7TB99) [reply doc/news/changes/minor/20240728AE7TB99]: I changed it back for consistency's sake.

**Comment by masterleinad** [comment]
> Indent check still does not pass, but I'm working on it...

Just squash all your commits and make sure the author information is correct then.

**Comment by masterleinad** [comment]
This pull request also makes some patches for the examples that we are using for testing invalid. We need to recreate them similarly and update them accordingly.

**Comment by tjhei** [comment]
Yeah, step-50 and step-49 we will need to handle separately because of ./tests/examples/.

@AE7TB99 Maybe it is easiest if you remove the changes to step-49 and step-50 for now so we can get this PR merged.

To make the changes in step-49 and 50 you will need to:
1. Compile the library without the step-49/50 changes.
2. call ``make setup_tests_examples`` then run ``ctest -R examples/``
3. update the examples.
4. go to ``build/tests/examples/`` and update the generated step-49 and step-50.cc files the same way you edited the original files.
5. run the "update_diffs" target as described in https://github.com/dealii/dealii/tree/master/tests/examples#example-testing
6. create a PR with the changes.

if you can not figure it out, let me know and I can do the changes for you.

**Review by masterleinad — APPROVED** [review]

**Pull request merged**